src/k8s/pkg/k8sd/setup/kube_apiserver_test.go

```diff
@@ -132,6 +132,19 @@ func TestKubeAPIServer(t *testing.T) {
 		g.Expect(len(args)).To(Equal(len(tests)))
 	})
 
+	t.Run("ArgsDualstack", func(t *testing.T) {
```

Comment:
I know this PR is addressing dualstack, but does ipv6 only also work? If so can we add a test case here?

Reply:
So, IPv6-only does not work with Cilium Loadbalancer so we don't use it (for now).
Apart from that IPv6-only should work. In this testcase we only check if the string-concatenated CIDRs are correctly passed down. Single CIDR is tested above (there is no difference between IPv6 and IPv4 CIDR since we just pass them down as strings.